lib/jira/resource/agile.rb

```diff
@@ -31,6 +31,21 @@ def self.get_sprint_issues(client, sprint_id, options = {})
         parse_json(response.body)
       end
 
+      def self.get_projects_full(client, board_id, options = {})
+        response = client.get(path_base(client) + "/board/#{board_id}/project/full")
+        parse_json(response.body)
+      end
+
+      def self.get_projects(client, board_id, options = {})
+        options[:maxResults] ||= 100
+        options[:startAt] ||= 0
+        create_meta_url = path_base(client) + "/board/#{board_id}/project"
+        params = hash_to_query_string(options)
+
+        response = params.empty? ? client.get("#{create_meta_url}") : client.get("#{create_meta_url}?#{params}")
```

Comment:
As you define default options, you will always have `params`, therefor you can simplify this line with:
```ruby
response = client.get("#{create_meta_url}?#{params}")
```